### ISSUE TITLE
Translate Sub-Zero

### ISSUE THREAD

**Issue by morpheus133** [pannal/Sub-Zero.bundle#458]
Hi, 
Is it planned to make translatable? 
Usually people use subtitle because they Dont understand English...

Onyl 3 thing needed:

- Add the translated texts to Sub-Zero.bundle\Contents\Strings\[lang].json
    eg:
    da.json
    de.json
    en.json
    es.json
    fr.json

- Add something like this to init_py Strat() methode:

```
def Start():
    HTTP.CacheTime = 0
    HTTP.Headers['User-agent'] = OS_PLEX_USERAGENT

    loc = GetLanguage()

    if Core.storage.file_exists(Core.storage.abs_path(
        Core.storage.join_path(
            Core.bundle_path,
            'Contents',
            'Strings',
            '%s.json' % loc
        )
    )):
        Locale.DefaultLocale = loc
    else:
        Locale.DefaultLocale = 'en-us'
...

def GetLanguage():
    return str(locale.getdefaultlocale()[0]).split('_')[0]
```

- Start some transifex or other translation thread and we can make the trasnlations.

I can help with the implementation if needed.

**Comment by pannal** [comment]
It's planned, yes, but there's an issue with translatable setting strings and Plex Web, which doesn't set the correct X-Plex-Language attribute when sending requests to plugins - at least last time I checked.

Once that issue is resolved, the translation implementation should be pretty straight forward, as everything already is implemented inside the PMS framework.

What do you need that specific locale setting for, exactly?

**Comment by morpheus133** [comment]
What is this issue?
I create one solution what seems to working:
[https://github.com/morpheus133/Sub-Zero.bundle/tree/translatesupporttest](https://github.com/morpheus133/Sub-Zero.bundle/tree/translatesupporttest)

I create two language: alpha  and beta (Just add "alpha" and "beta" to each string )
And one new options where we can change the language. It seems working.
(Maybe a better solution to use plex config language as deafult, but I dont find it in API where to get it.)

alpha:
![image](https://user-images.githubusercontent.com/6975928/37150002-653f3116-22d0-11e8-968f-b04012c103c5.png)

beta:
![image](https://user-images.githubusercontent.com/6975928/37150054-8a83ffa6-22d0-11e8-8182-1172f70e2d49.png)

**Comment by pannal** [comment]
Yeah I know that that's forcable. The issue is, that the language should be automatically changed depending on the user's Plex Web language setting.

There's an internal issue open for Plex Web, that it should pass the X-Plex-Language header.

**Comment by pannal** [comment]
We will be using POEditor.com for SZ's translation. Join here: http://v.ht/szi18n
